Makefile

```diff
@@ -12,7 +12,7 @@ kafka_$(KAFKA_VERSION)_amd64.snap:
 
 .PHONY: fat-charm
 fat-charm: kafka_$(KAFKA_VERSION)_amd64.snap
-	cp $< charm/kafka
+	cp $< charm/kafka/kafka.snap
```

Comment:
OK to rename it, but if we're giving the snap a fixed filename, that can replace the wildcard matching in install_snap below. I believe the resource also gets a fixed filename.